apps/web/src/pages/get-started/components/CodeSnippet.tsx

```diff
@@ -0,0 +1,45 @@
+import { Input, inputStyles } from '@novu/design-system';
+import { useClipboard } from '@mantine/hooks';
+import { css, cx } from '../../../styled-system/css';
+import { ClipboardIconButton } from '../../../components';
+
+const codeValueInputClassName = css({
+  '& input': {
+    border: 'none !important',
+    background: 'surface.popover !important',
+    color: 'typography.text.secondary !important',
+    fontFamily: 'mono !important',
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** @davidsoderberg Updated this component to use the tokens we already have!

We had a similar functionality in the Information Architecture changes, so I updated the file to leverage that‏